app/src/main/java/ip/team13/petowner/ui/activities/models/ActivityData.kt

```diff
@@ -0,0 +1,42 @@
+package ip.team13.petowner.ui.activities.models
```

Comment:
Can we keep the `models` inside the `data` package? It seems this file actually belongs to the domain models